### PR TITLE
fix(config-generator): fix support for Prettier config files

### PR DESCRIPTION
**Pull request by TefDesign**
| Q             | A
| ------------- | ---
| Bug fix?      | yes

Accept many config file type for prettier config. List of accepted files : 
- .prettierrc.js
- .prettierrc.cjs
- prettier.config.js
- prettier.config.cjs

### CODE REVIEW

File: packages/eslint-config-generator/src/config-generator.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ function getBaseConfig(options: Options): ESLintConfig {
             '**/postcss.config.{js,ts}', // postcss config
             '**/tailwind.config.{js,ts}', // tailwind config
             '**/vite.config.{js,ts}', // vite config
-            '**/prettier.config.{js,ts}', // prettier config
+            '**/*prettier{.config, rc}.{js,ts,cjs}', // prettier config
```
Comment (Kocal) [inline packages/eslint-config-generator/src/config-generator.ts:86]:
```suggestion
            '**/prettier.config.{js,cjs,ts}', // prettier config
            '**/.prettierrc.{js,cjs,ts}', // prettier config
```

Easier to read and maintain IMO

Comment (TefDesign) [reply packages/eslint-config-generator/src/config-generator.ts]: Done @Kocal ;)

File: common/changes/@kocal/eslint-config-generator/fix-config-file-prettier_2022-02-18-21-13.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@kocal/eslint-config-generator",
+      "comment": "Accept many config file type for prettier config. List of accepted files : .prettierrc.js, .prettierrc.cjs, prettier.config.js et prettier.config.cjs",
```
Comment (Kocal) [inline common/changes/@kocal/eslint-config-generator/fix-config-file-prettier_2022-02-18-21-13.json:5]:
```suggestion
      "comment": "Fixed supported Prettier configuration files, .prettierrc.* files are now also supported.",
```

File: packages/eslint-config-generator/src/config-generator.test.ts [hunk 3]
```diff
@@ -63,7 +63,8 @@ describe('Configuration generator', function () {
               '**/postcss.config.{js,ts}',
               '**/tailwind.config.{js,ts}',
               '**/vite.config.{js,ts}',
-              '**/prettier.config.{js,ts}',
+              '**/prettier.config.{js,cjs,ts}',
+              '**/.prettierrc.{js,cjs,ts}',
```
Comment (Kocal) [inline packages/eslint-config-generator/src/config-generator.test.ts:67]: In fact it looks like Prettier does not support `.ts` files, see https://prettier.io/docs/en/configuration.html

So you can remove `,ts` for both entries, thanks.

**Review by Kocal — CHANGES_REQUESTED** [review]

**Comment by Kocal** [comment]
Please rebase your branche

File: packages/eslint-config-generator/src/config-generator.test.ts (hunk 3 above)
Comment (TefDesign) [reply packages/eslint-config-generator/src/config-generator.test.ts]: done

File: common/changes/@kocal/eslint-config-generator/fix-config-file-prettier_2022-02-18-21-13.json (hunk 2 above)
Comment (TefDesign) [reply common/changes/@kocal/eslint-config-generator/fix-config-file-prettier_2022-02-18-21-13.json]: done

**Comment by TefDesign** [comment]
> Please rebase your branch

done

**Comment by Kocal** [comment]
Thanks!

**Pull request merged**